mmengine/model/utils.py

```diff
@@ -195,3 +195,49 @@ def revert_sync_batchnorm(module: nn.Module) -> nn.Module:
         module_output.add_module(name, revert_sync_batchnorm(child))
     del module
     return module_output
+
+
+def convert_sync_batchnorm(module: nn.Module, sync_bn='torch') -> nn.Module:
+    """Helper function to convert all `BatchNorm` layers in the model to
+    `SyncBatchNorm` (SyncBN) or `mmcv.ops.sync_bn.SyncBatchNorm`(MMSyncBN)
+    layers. Adapted from <https://pytorch.org/docs/stable/generated/torch.nn.Sy
+    ncBatchNorm.html#torch.nn.SyncBatchNorm.convert_sync_batchnorm>_.
+
+    Args:
+        module (nn.Module): The module containing `SyncBatchNorm` layers.
+        sync_bn (str): The type of `SyncBatchNorm` to convert to.
+
+            - 'torch': convert to `torch.nn.modules.batchnorm.SyncBatchNorm`.
+            - 'mmcv': convert to `mmcv.ops.sync_bn.SyncBatchNorm`.
+
+    Returns:
+        nn.Module: The converted module with `SyncBatchNorm` layers.
+    """  # noqa: E501
+    module_output = module
+
+    if isinstance(module, torch.nn.modules.batchnorm._BatchNorm):
+        if sync_bn == 'torch':
+            SyncBatchNorm = torch.nn.modules.batchnorm.SyncBatchNorm
+        elif sync_bn == 'mmcv':
+            from mmcv.ops import SyncBatchNorm  # type: ignore
+        else:
+            raise ValueError('sync_bn should be "torch" or "mmcv", but got '
```

Comment:
The outer parameter name is `convert_sync_bn,` and the inner one is `sync_bn`, it will be difficult for users to understand. It is recommended to keep it the same.

sync_bn should be torch or mmcv but got {}. Please specify the `cfg.convert_sync_bn` parameter correctly